app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -193,47 +214,108 @@ You will not be eligible for the IRP, because you have spent more than 90 days i
 
 You travel to attend an interview for an eligible teaching job in the UK under a Standard Visitor visa in May 2024. The visit lasts 2 days and you return home, and learn you have been successful in your application. The role begins in September 2024. You apply for a Skilled Worker visa in June 2024, 3 months before your job begins. You move to the UK in July 2024.
 
-You will be eligible for the IRP, because you have spent less than 90 days in the UK in the previous 12 months and have moved to the UK less than 3 months before the start of your teaching job.
+You will be eligible for the first instalment of IRP, because you have spent less than 90 days in the UK in the previous 12 months and have moved to the UK less than 3 months before the start of your teaching job.
 
 #### Example 3:
 
 You study in the UK on a Student visa for three years from September 2019 to July 2022, before returning home. You apply for an eligible teaching job in England starting in September 2024. You apply for a Skilled Worker visa in June 2024, 3 months before the start of your job. You move to the UK in July 2024.
 
-You will be eligible for the IRP because you have spent less than 90 days in the UK in the previous 12 months and have moved to the UK less than 3 months before the start of your job.
+You will be eligible for the first installment of IRP because you have spent less than 90 days in the UK in the previous 12 months and have moved to the UK less than 3 months before the start of your job.
 
-### Non-eligibility
+### Eligibility requirements for IRP instalment 2 (£5,000)
 
-You will not be eligible for the international relocation payment if you are:
+To get the second instalment of the IRP, you must meet all the eligibility requirements set out in this section.
 
-* a UK citizen, including citizens of Wales, Scotland and Northern Ireland
+#### Teaching subject and school
 
-* an Irish citizen
+To be eligible for the second instalment of the IRP, you must:
 
-* a non-UK citizen who moved to England more than 3 months before the start of your job
+* have worked as a teacher for the previous year, that is, 3 consecutive terms
```

Comment:
```suggestion
* have worked as a teacher for the previous year (3 consecutive terms)
```
I find the `that is` a bit confusing - is there a better way of doing this? But ignore if needed!

Reply:
is it worth saying academic terms? is this a common term that non-UK citizens would understand?

Reply:
I'll change to your suggestion as this reads much better (have tweaked slightly), will need to check with policy as Amy said it needed to be worded in a certain way.